[2023-08-31 13:22:09,456] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-09-24T00:00:00+00:00 [queued]>
[2023-08-31 13:22:09,558] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-09-24T00:00:00+00:00 [queued]>
[2023-08-31 13:22:09,559] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:22:09,560] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-08-31 13:22:09,560] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:22:09,615] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-09-24T00:00:00+00:00
[2023-08-31 13:22:09,625] {standard_task_runner.py:52} INFO - Started process 2799 to run task
[2023-08-31 13:22:09,646] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2021-09-24T00:00:00+00:00', '--job-id', '174', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmphnltk6e0', '--error-file', '/tmp/tmp76883lli']
[2023-08-31 13:22:09,650] {standard_task_runner.py:77} INFO - Job 174: Subtask second_task
[2023-08-31 13:22:09,835] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2021-09-24T00:00:00+00:00 [running]> on host e7718740c1e6
[2023-08-31 13:22:09,996] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=devendra
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-24T00:00:00+00:00
[2023-08-31 13:22:09,997] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-08-31 13:22:09,999] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2023-08-31 13:22:10,041] {bash.py:169} INFO - Output:
[2023-08-31 13:22:10,102] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2023-08-31 13:22:10,108] {bash.py:177} INFO - Command exited with return code 0
[2023-08-31 13:22:10,160] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210924T000000, start_date=20230831T132209, end_date=20230831T132210
[2023-08-31 13:22:10,285] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-31 13:22:10,344] {local_task_job.py:146} INFO - Task exited with return code 0
